### PR TITLE
Fix cli runner: use embed_metadata_in_job parameter

### PR DESCRIPTION
**Pull request by ThomasWollmann**
Added code for reading the DEFAULT_EMBED_METADATA_IN_JOB parameter in jobs/runners/cli.py job submission.

### CODE REVIEW

**Comment by nsoranzo** [comment]
@ThomasWollmann Can you please fix the errors reported in https://travis-ci.org/galaxyproject/galaxy/jobs/121647820 ? Thanks!

**Comment by ThomasWollmann** [comment]
Related to: #1894

File: lib/galaxy/jobs/runners/cli.py [hunk 1]
```diff
@@ -50,8 +52,9 @@ def parse_destination_params( self, params ):
     def queue_job( self, job_wrapper ):
         """Create job script and submit it to the DRM"""
         # prepare the job
-        if not self.prepare_job( job_wrapper, include_metadata=True ):
-            return
+        include_metadata = asbool( job_wrapper.job_destination.params.get( "embed_metadata_in_job", DEFAULT_EMBED_METADATA_IN_JOB ) )
+        if not self.prepare_job( job_wrapper, include_metadata=include_metadata):
+             return
```
Comment (nsoranzo) [inline lib/galaxy/jobs/runners/cli.py]: Here `return` is indented with 5 chars with respect to the line above, should be 4.

**Comment by nsoranzo** [comment]
@galaxybot test this

**Comment by jmchilton** [comment]
Wonderful - thanks a million for the fix/enhancement.

**Comment by bgruening** [comment]
Awesome! Thanks @ThomasWollmann! Hope you enjoyed the Workshop!

**Pull request merged**